### PR TITLE
 Introduce more taur clothes.

### PR DESCRIPTION
**Pull request by lbnesquik**
- Add medieval style barding for centaurs.
- Add full body options for wolf taurs. (Fat sprites slightly break this)
- Cloak for Drakes
- Botany Apron should work for the majority of taurs and not weirdly cut at the middle.
- WIP lance

### CODE REVIEW

**Comment by lbnesquik** [comment]
aseprite_2019-11-02_10-55-40
dreamseeker_2019-11-01_02-46-18
aseprite_2019-10-29_20-37-59

**Comment by lbnesquik** [comment]
Heck. Conflicts. Should i pull from master to try and fix or?

**Comment by TheFurryFeline** [comment]
> Heck. Conflicts. Should i pull from master to try and fix or?

That's how you tend to fix conflicts, yeah.

**Comment by TheFurryFeline** [comment]
Funny thing is, I was the one who recently changed vending_vr.dm for a brig variation of seed vendor and NutriMax.

**Comment by TheFurryFeline** [comment]
Oh, I see what's wrong here. Lots of icons have been deleted by an outdated branch. https://github.com/VOREStation/VOREStation/pull/6367/checks?check_run_id=363389007

**Comment by lbnesquik** [comment]
@Novacat This should be good to go.

File: code/game/objects/items/weapons/material/twohanded.dm [hunk 1]
```diff
@@ -157,3 +157,18 @@
 	fragile = 1	//It's a haphazard thing of glass, wire, and steel
 	reach = 2 // Spears are long.
 	attackspeed = 14
+
+//This is mostly for centaurs.
+/obj/item/weapon/material/twohanded/spear/lance
+	name = "lance"
+	desc = "End him rightly"
```
Comment (KasparoVy) [inline code/game/objects/items/weapons/material/twohanded.dm]: Gonna need a pommel for that one.

Comment (lbnesquik) [reply code/game/objects/items/weapons/material/twohanded.dm]: Like, a jousting lance? Is that not the correct term for that?

Comment (KasparoVy) [reply code/game/objects/items/weapons/material/twohanded.dm]: Sorry, I was mostly joking -- referring to https://knowyourmeme.com/memes/end-him-rightly, popularized by a guy named Skallagrim

But yeah a pommel is a piece from the bottom of a sword's grip, not quite a lance

**Comment by TheFurryFeline** [comment]
Yay, conflicts resolved. Now i need to figure out how to fix that sort of dmi conflict on another PR on Polaris, bleh. My attempts to do it resulted in conflicts happening. Hopefully will get granted repo access to the op's PR's.

**Pull request merged**